src/shared/components/person/reports.tsx

```diff
@@ -610,16 +611,28 @@ export class Reports extends Component<any, ReportsState> {
   async handleResolveCommentReport(form: ResolveCommentReport) {
     const res = await HttpService.client.resolveCommentReport(form);
     this.findAndUpdateCommentReport(res);
+    if (this.state.unreadOrAll === UnreadOrAll.Unread) {
+      this.refetch();
```

Comment:
Wouldn't this need to await?

Reply:
I see no problem in refetching and rendering the reports at the same time as the unread counts are refreshed. The resolveComment call has already awaited so the data will be fresh for both calls and the UnreadCounterService will sooner be refreshed and doesn't need to wait for the rest of the UI to update

Reply:
Ah okay.